iac-common/src/test/java/org/sonar/iac/common/testing/ExtensionSensorTest.java

```diff
@@ -74,12 +75,16 @@ void shouldRaiseIssueOnParsingErrorWhenIssueActive() {
     assertThat(analysisError.message()).startsWith("Unable to parse file:");
 
     // Test logging
-    assertThat(logTester.logs(LoggerLevel.ERROR)).hasSize(2);
-    assertThat(logTester.logs(LoggerLevel.ERROR)).extracting(s -> s)
-      .anyMatch(s -> s.startsWith(String.format("Unable to parse file: %s.", inputFile.uri())))
-      .anyMatch(s -> s.startsWith(String.format("Cannot parse '%s':", inputFile.filename())));
+    assertThat(logTester.logs(LoggerLevel.ERROR)).hasSize(1);
+    String errorMessage = "Cannot parse '%s:1:1'";
+    assertThat(logTester.logs(LoggerLevel.ERROR).get(0))
+      .startsWith(String.format(errorMessage, inputFile.filename()));
+
+    verifyDebugMessages(logTester.logs(LoggerLevel.DEBUG));
   }
 
+  protected abstract void verifyDebugMessages(List<String> logs);
+
```

Comment:
I would move this method declaration with the other, at the beginning of the class declaration.